app/src/main/java/org/mozilla/fenix/utils/Settings.kt

```diff
@@ -39,7 +40,8 @@ class Settings private constructor(context: Context) {
         get() = preferences.getString(appContext.getPreferenceKey(R.string.pref_key_search_engine), "") ?: ""
 
     val isCrashReportingEnabled: Boolean
-        get() = preferences.getBoolean(appContext.getPreferenceKey(R.string.pref_key_crash_reporter), true)
+        get() = preferences.getBoolean(appContext.getPreferenceKey(R.string.pref_key_crash_reporter), true) &&
```

Comment:
So we disable crash reporting with BUILD_TYPE == "release"?

Reply:
Good catch on this. Missed that when moving it from the early return condition!